cli.js

```diff
@@ -2,8 +2,8 @@
 
 'use strict'
 
-var nodeVersionInfo = process.versions.node.split('.').map(function (n) { return Number(n) })
-if (nodeVersionInfo < [4, 0, 0]) {
+var nodeMajorVersion = Number(process.versions.node.split('.')[0])
+if (nodeMajorVersion < 4) {
```

Comment:
If you want to check if the version is greater than some specific semver in the future, I would require `semver` and use its comparison method... inequality checks on arrays convert to strings ... so `[4,2,0] > [4,10,0] == true` but should be `false` when comparing two versions.  

But, if all this cares about is the major version, then this approach is sufficient.